src/BgZ-3-0/Cert/_reference/resources/medmij-bgz-medicationagreement-ts-01.xml

```diff
@@ -42,7 +42,7 @@
          <extension url="http://nictiz.nl/fhir/StructureDefinition/practitionerrole-reference">
             <valueReference>
                <reference value="PractitionerRole/medmij-bgz-practitionerrole-ts-04"/>
-               <display value="Peter van Pulver"/>
+               <display value="Huisartsen"/>
```

Comment:
A similar construction is present in the MedicationUse fixture, could you change that one as well? Moreover, in other places in the BgZ (i.e. in the Vaccination and Patient fixtures) the `.display` for a general practitioner reference is _Huisartsen, niet nader gespecificeerd_, so I think it's better to make it consistent everywhere.